src/Particular.CodeRules/tools/install.ps1

```diff
@@ -0,0 +1,6 @@
+﻿param($installPath, $toolsPath, $package, $project)
```

Comment:
Afaik instal.ps1 is only executed on install. Not restore. Init.ps1 is run when the solution is open but only if the package manager is open at that time